packages/material-ui/src/Box/Box.d.ts

```diff
@@ -14,9 +15,425 @@ import {
   PropsFor,
 } from '@material-ui/system';
 import { Theme } from '../styles/createMuiTheme';
-import { CSSObject } from '../styles/experimentalStyled';
 import { Omit } from '..';
 
+/**
+ * CSS typings
+ */
+
+export type StandardCSSProperties = CSS.PropertiesFallback<number | string>;
+
+/**
+ * The `css` function accepts arrays as values for mobile-first responsive styles.
+ * Note that this extends to non-theme values also. For example `display=['none', 'block']`
+ * will also works.
+ *
+ * For more information see: https://styled-system.com/responsive-styles
+ */
+export type ResponsiveStyleValue<T> = T | Array<T | null> | { [key: string]: T | null };
+
+/**
+ * All non-vendor-prefixed CSS properties. (Allow `number` to support CSS-in-JS libs,
+ * since they are converted to pixels)
+ */
+export interface CSSProperties
+  extends CSS.StandardProperties<number | string>,
+    CSS.SvgProperties<number | string> {}
+
+/**
+ * Map of all CSS pseudo selectors (`:hover`, `:focus`, ...)
+ */
+export type CSSPseudoSelectorProps = { [K in CSS.Pseudos]?: SystemStyleObject };
+
+/**
+ * CSS as POJO that is compatible with CSS-in-JS libaries.
```

Comment:
```suggestion
 * CSS as a plain object that is compatible with CSS-in-JS libaries.
```
Java can keep their jargon :wink: